docs/.link-check-errors.txt

```diff
@@ -1,5 +1,11 @@
 Broken links found by `docs:link-check` that exist:
 
+- https://mui.com/x/react-charts/getting-started/
 - https://mui.com/x/react-data-grid/#premium-plan
+- https://mui.com/x/react-data-grid/getting-started/
 - https://mui.com/x/react-data-grid/getting-started/#feature-comparison
 - https://mui.com/x/react-date-pickers/fields/#accessible-dom-structure
+- https://mui.com/x/react-date-pickers/getting-started/
+- https://mui.com/x/react-tree-view/#rich-tree-view
+- https://mui.com/x/react-tree-view/#simple-tree-view
+- https://mui.com/x/react-tree-view/getting-started/
```

Comment:
As I understand things, we should fix those

```suggestion
```

Reply:
They're all coming from the Material UI docs and MUI blog - PR to fix them here: https://github.com/mui/material-ui/pull/45145

Reply:
Ah, so it's hiding something else. So same fix as https://github.com/mui/base-ui/pull/552 and we are good,